add-on/chrome/bootstrap.js

```diff
@@ -469,6 +469,11 @@ var WindowListener = {
           // Do the normal behavior first.
           controller._doCommand.apply(controller, arguments);
 
+          // Don't bother prompting the user if already sharing.
+          if (this.MozLoopService.screenShareActive) {
```

Comment:
No need to check `ticket` as `addCopyPanel` only gets called if ticket passes throttling checks, so `ticket` will be 255. I did ask about not showing the panel if the user has ever successfully created a room, and it wasn't written in the bug, but it was explicitly mentioned to only check for an active session to prevent copy panel and not previous rooms.